howso/engine/trainee.py

```diff
@@ -4037,7 +4056,7 @@ def load_trainee(
     # file name.
     if file_path.suffix:
         # Check to make sure sure `.caml` file is provided
-        if file_path.suffix.lower() != '.caml':
+        if file_path.suffix.lower() not in ('.caml', '.amlg'):
             raise HowsoError(
                 'Filepath with a non `.caml` extension was provided.'
```

Comment:
Need to update error text to also include `amlg`

Reply:
My read of the code originally is that there was a conscious intent to not allow uncompressed `.amlg` files, and maybe at this point the right fix is to put it back to compressed `.caml` only.